omod/src/main/java/org/openmrs/module/coreapps/page/controller/MarkPatientDeadPageController.java

```diff
@@ -69,5 +84,16 @@ public String post(@SpringBean("patientService") PatientService patientService,
         }
         return conceptAnswers;
     }
+
+    /**
+     * This Method closes all active visits when a patient dies.
+     */
+    private void actionAfterDeath(Patient patient, Date deathDate) {
+        VisitService visitService = Context.getVisitService();
+        List<Visit> visitList = visitService.getActiveVisitsByPatient(patient);
```

Comment:
Can you please make aboe two lines to one line?

Reply:
Will do